frontend/src/components/UserCard.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { Button } from "semantic-ui-react";
+import { useAuthContext } from "../contexts/AuthContext";
+
+const UserCard = () => {
+  const { user, signOut } = useAuthContext();
+
+  // css styles
+  const userCardStyles = `container flex flex-col p-6 mx-auto justify-center items-center 
+    space-y-4 rounded-xl`;
+  const buttonStyles = ``;
+
+  const handleSignOut = async () => {
+    console.log("Signing out...");
+    const result = await signOut();
+    console.log(result);
+  };
+  return (
+    <div className={`${userCardStyles}`} id="user-card">
+      {user && user.emailVerified ? (
+        <>
```

Comment:
Wrap views into its own components to make things tidy.

Reply:
I'm not sure what you mean here. So instead of handling the render logic within this component I should have a component that renders the user card and a component that doesn't and return one of the two?

Reply:
Something like 
```
user && user.emailVerified ? <UserNav/>:<DefaultNav/>
```